modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -298,6 +303,23 @@ public void testGetRelatedCustomObjectsWhenRelationExists()
 
 		_assertEquals(_objectEntry2, jsonObject.getJSONArray("items"));
 
+		ObjectRelationshipTestUtil.relateObjectEntries(
```

Comment:
What about adding a specific test for pagination instead?

Otherwise, this test is a bit weird as first we test the happy path with 1 object entry in a relationship, and then, we relate with a new object entry and test the pagination

Reply:
Will do.